src/base/utils/misc.cpp

```diff
@@ -115,12 +112,20 @@ void Utils::Misc::shutdownComputer(const ShutdownDialogAction &action)
     if (GetLastError() != ERROR_SUCCESS)
         return;
 
-    if (action == ShutdownDialogAction::Suspend)
-        SetSuspendState(false, false, false);
-    else if (action == ShutdownDialogAction::Hibernate)
-        SetSuspendState(true, false, false);
-    else
+    using PSETSUSPENDSTATE = BOOLEAN (WINAPI *)(BOOLEAN, BOOLEAN, BOOLEAN);
+    const auto setSuspendState = Utils::Misc::loadAPI<PSETSUSPENDSTATE>("PowrProf.dll", "SetSuspendState");
+
+    if (action == ShutdownDialogAction::Suspend) {
+        if (setSuspendState)
+            setSuspendState(false, false, false);
+    }
+    else if (action == ShutdownDialogAction::Hibernate) {
+        if (setSuspendState)
+            setSuspendState(true, false, false);
+    }
+    else {
         InitiateSystemShutdownA(0, QCoreApplication::translate("misc", "qBittorrent will shutdown the computer now because all downloads are complete.").toLocal8Bit().data(), 10, true, false);
```

Comment:
Since you're here, could you change it with InitiateSystemShutdownW?

Reply:
both done.

Reply:
It's still unchanged...